crates/figma_import/src/reflection.rs

```diff
@@ -21,248 +20,10 @@ pub fn registry() -> serde_reflection::Result<serde_reflection::Registry> {
-    tracer
-        .trace_type::<dc_bundle::definition::element::variable_value::Value>(&samples)
-        .expect("couldn't trace variable_value::Value");
-    tracer
-        .trace_type::<dc_bundle::definition::element::Variable>(&samples)
-        .expect("couldn't trace Variable");
-    tracer
-        .trace_type::<dc_bundle::definition::element::VariableMap>(&samples)
-        .expect("couldn't trace VariableMap");
-    tracer
-        .trace_type::<dc_bundle::definition::element::ViewShape>(&samples)
-        .expect("couldn't trace ViewShape");
-    tracer
-        .trace_type::<dc_bundle::definition::element::view_shape::Shape>(&samples)
-        .expect("couldn't trace Shape");
-    tracer
-        .trace_type::<dc_bundle::definition::view::ViewData>(&samples)
-        .expect("couldn't trace ViewData");
-    tracer
-        .trace_type::<dc_bundle::definition::view::view_data::ViewDataType>(&samples)
-        .expect("couldn't trace ViewDataType");
-    tracer
-        .trace_type::<dc_bundle::definition::view::view_data::StyledTextRuns>(&samples)
-        .expect("couldn't trace StyledTextRuns");
-    tracer
-        .trace_type::<dc_bundle::definition::view::view_data::Container>(&samples)
-        .expect("couldn't trace Container");
-    tracer
-        .trace_type::<dc_bundle::definition::view::view_data::Text>(&samples)
-        .expect("couldn't trace Text");
-    tracer.trace_type::<dc_bundle::definition::view::View>(&samples).expect("couldn't trace View");
-    tracer.trace_type::<layout::types::Layout>(&samples).expect("couldn't trace Layout");
-    tracer
-        .trace_type::<layout::layout_manager::LayoutChangedResponse>(&samples)
-        .expect("couldn't trace LayoutChangedResponse");
-    tracer
-        .trace_type::<layout::layout_node::LayoutNode>(&samples)
-        .expect("couldn't trace LayoutNode");
-    tracer
-        .trace_type::<layout::layout_node::LayoutNodeList>(&samples)
-        .expect("couldn't trace LayoutNodeList");
-    tracer.trace_type::<EncodedImageMap>(&samples).expect("couldn't trace EncodedImageMap");
-    tracer.trace_type::<crate::NodeQuery>(&samples).expect("couldn't trace NodeQuery");
-    tracer
-        .trace_type::<crate::DesignComposeDefinitionHeader>(&samples)
-        .expect("couldn't trace SerializedDesignDocHeader");
-    tracer
-        .trace_type::<crate::DesignComposeDefinition>(&samples)
-        .expect("couldn't trace SerializedDesignDoc");
-    tracer.trace_type::<crate::ServerFigmaDoc>(&samples).expect("couldn't trace ServerFigmaDoc");
+    // tracer.trace_type::<EncodedImageMap>(&samples).expect("couldn't trace EncodedImageMap");
+    // tracer.trace_type::<crate::NodeQuery>(&samples).expect("couldn't trace NodeQuery");
+    // tracer.trace_type::<crate::ServerFigmaDoc>(&samples).expect("couldn't trace ServerFigmaDoc");
     tracer.trace_type::<crate::ConvertResponse>(&samples).expect("couldn't trace ConvertResponse");
```

Comment:
We still need this one? Is this just a TODO for later to convert it to proto?

Reply:
It's in the description but I added a comment too. We'll clean it up sometime soon but it's not blocking us from finishing the rest of the proto conversion.